init-tools.msbuild

```diff
@@ -8,5 +8,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.DotNet.BuildTools" Version="$(BuildToolsPackageVersion)" />
     <PackageReference Include="Microsoft.DotNet.BuildTools.Coreclr" Version="1.0.4-prerelease" />
+    <PackageReference Include="ILLink.Tasks" Version="0.1.5-preview-1461378" />
```

Comment:
I'll add this to `dependencies.props` in my other PR (where I introduce `dependencies.props`) to match https://github.com/dotnet/coreclr/blob/2e5f43401b458315dc33c137f0ee0c5a02d810c2/init-tools.msbuild#L20-L22.